modules/apps/frontend-js/frontend-js-react-web/types/src/main/resources/META-INF/resources/js/accessibilityMenu.d.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
@matuzalemsteles this feels weird to add this to frontend-js-react-web, right? Is there a precedence for this type of inclusion?

Reply:
@ethib137 yeah but unfortunately that was the only way i found to fix the circular dependency problem, i added a comment about it [#122 (comment)](https://github.com/liferay-peds/liferay-portal/pull/122#issuecomment-1615225431), I think the amount of comment hid it.

Basically, the error it was causing in the tests is related to the circular dependency problem due to the `document-library-opener-onedrive-web` module -> `frontend-js-web` -> `frontend-js-react-web` -> `accessibility-menu-web` -> `cookies-banner-web` -> `frontend-js-web`. Also the cookies-banner-web module was [causing some strange test failures](https://github.com/liferay-peds/liferay-portal/pull/122#issuecomment-1615263602) due to turning the module to ESM.